src/javascript/app_2/App/Components/Layout/Header/menu_links.jsx

```diff
@@ -1,52 +1,28 @@
 import PropTypes      from 'prop-types';
 import React          from 'react';
-import InkBar         from '../../Elements/InkBar';
+import withInkBar     from '../../Elements/InkBar';
 import { BinaryLink } from '../../Routes';
 import Url            from '../../../../../_common/url';
 
-class MenuLinks extends React.PureComponent {
-    state = {
-        width: 0,
-        left : 0,
-    };
+// components wrapped with withInkBar has to be outside of render
+const DivWithInkBar = withInkBar((props) => <div {...props} />);
```

Comment:
@kellybinary 
is it possible to use other kinds of components instead of `div` or it's the allowable one?

Reply:
yes, it's possible to use other kinds of components as well

Reply:
so why isn't it passed directly to the `ComponentWithInkBar` as a prop? Does it cause rendering issues?